govuk_publishing_components.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.homepage    = "https://github.com/alphagov/govuk_publishing_components"
   s.license     = "MIT"
 
-  s.files = Dir["{node_modules/govuk-frontend,app,config,db,lib}/**/*", "DEVELOPMENT.md", "LICENCE.md", "Rakefile", "README.md"]
+  s.files = Dir["{node_modules/govuk-frontend,node_modules/axe-core,app,config,db,lib}/**/*", "DEVELOPMENT.md", "LICENCE.md", "Rakefile", "README.md"]
```

Comment:
Come to think of it, should this be the same as https://github.com/alphagov/govuk_publishing_components/blob/master/package.json#L16-L18? So jquery as well?

In that case we can automate it with something like 

```rb
JSON.parse(File.read("package.json"))["dependencies"].keys
```

Reply:
I was thinking it might also need jquery, yes. Thanks - probably going to end up doing a few versions of the gem this afternoon!

Where would the example line you gave go?

Reply:
Seems to have worked without jquery...